scripts/backend_tests.py

```diff
@@ -215,6 +220,12 @@ def _convert_to_test_target(path):
             result.append(_convert_to_test_target(
                 os.path.join(base_path, root)))
         for subroot, _, files in os.walk(os.path.join(base_path, root)):
+            if subroot.endswith(_LOAD_TESTS_DIR) and include_load_tests:
```

Comment:
If we're going to have subdirectories in that test dir, should this be something along the lines of "subroot contains load_test_dir" rather than "ends with"? (I'm not sure.)